app/src/main/res/layout/item_timeline_conversation.xml

```diff
@@ -57,8 +57,10 @@
                     android:id="@+id/image_first_audio"
                     android:layout_width="@dimen/dp_24"
                     android:layout_height="@dimen/dp_24"
-                    android:background="@drawable/ic_audio_conversation"
+                    android:background="@drawable/bg_menu_media_creating_conversation"
                     android:contentDescription="@string/description_recording_audio_convention"
+                    android:padding="@dimen/dp_4"
+                    android:src="@drawable/ic_menu_media"
```

Comment:
ERROR: [VectorDrawableCompat] When using VectorDrawableCompat, you need to use `app:srcCompat`.
 To use VectorDrawableCompat, you need to make two modifications to your project. First, set `android.defaultConfig.vectorDrawables.useSupportLibrary = true` in your `build.gradle` file, and second, use `app:srcCompat` instead of `android:src` to refer to vector drawables.